src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public Excavator(String name) {
+        super(name);
+    }
```

Comment:
if we have this constructor in Parent class , should we create it in Child classes?